docker/src/main/java/org/jclouds/docker/compute/options/DockerTemplateOptions.java

```diff
@@ -384,7 +400,7 @@ public static DockerTemplateOptions overrideLoginCredentials(LoginCredentials cr
       }
 
       /**
-       * @see TemplateOptions#blockUntilRunning
+       * @see TemplateOptions#blockUntilRunning(boolean)
        */
       public static DockerTemplateOptions blockUntilRunning(boolean blockUntilRunning) {
          DockerTemplateOptions options = new DockerTemplateOptions();
```

Comment:
What is the purpose of having the options builder methods accepting a `null` value? Is there a need to "clear" any previously set value?

BTW, having a look at this class, it makes no sense to have the variables being an Optional. They're only used internally so they could be perfectly plain nullable objects. That would allow to cleanup the compute service adapter code and make it more readable. WDYT?